File: Code Chef/JavaScript/700-799/772 Janmansh and Games JGAMES.js
let input = '';
process.stdin.on('data', (data) => input += data);

const strToInt = (strArray) => {
    strArray.shift();
    const intArray = strArray.map((strItem) => strItem.trim().split(' ').map(Number));
    
    return intArray;
};

const whoWillWin = (num, moves) => {
    const maxScore = num + moves;
    const isEven = (maxScore % 2) === 0;
    
    const winner = (isEven) ? 'Janmansh' : 'Jay';
    console.log(winner);
};

process.stdin.on('end', () => {
    const testcases = strToInt(input.trim().split('\n'));
    
    testcases.forEach((testcase) => {
        const [num, moves] = testcase;
        whoWillWin(num, moves);
    });
});

File: Code Chef/C Programming/700-799/772 Janmansh and Games JGAMES.c
#include <stdio.h>
int main(void)
{
	int t,x,y;
	scanf("%d",&t);
	for(int i=0;i<t;i++)
	{
	    scanf("%d %d",&x,&y);
	    if(((x+y)%2)==0)
	    {
	        printf("Janmansh\n");
	    }
	    else
	    {
	        printf("Jay\n");
	    }
	}
	return 0;
}